### PR TITLE
[nrf noup] workflows: Update the commit tags action

### PR DESCRIPTION
**Pull request by carlescufi**
nrf-squash! [nrf noup] ci: NCS-specific CI tweaks

Adapt to the changes in:
https://github.com/nrfconnect/action-commit-tags/pull/4

### CODE REVIEW

**Comment by carlescufi** [comment]
@nrfconnect/ncs-code-owners can I get reviews?

I will switch the revision from `pull/4/head` to `main` after this is approved and ready to merge.

**Review by nordicjm — APPROVED** [review]

**Review by bjarki-andreasen — APPROVED** [review]

**Pull request merged**
